types/utils.d.ts

```diff
@@ -24,6 +27,10 @@ export type ObjectOrFunc<TResult, TArg = {}> = ((arg: TArg) => TResult) | TResul
 
 export type Props = ObjectOf<any>
 export type ReactChildren = React.ReactNodeArray | React.ReactNode
+
+export type ReactPropsStrict<T> = { [K in keyof T]: NullableIfUndefined<T[K]> }
+export type ReactProps<T> = Extendable<ReactPropsStrict<T>>
```

Comment:
So when I should use `Extendable` vs `ReactProps`? My understanding is I should always use `ReactProps` (or `ReactPropsStrict` in special cases).
But for example `Dropdown` is still using `Extendable`.
Shouldn't we remove `Extendable` completely?

Reply:
`Dropdown` should use `ReactProps`, too. I missed it, will check all `Dropdown`'s components.

> Shouldn't we remove Extendable completely?

Note sure, it's a quite generic type, we're using it in `theme/types.ts` for colors and props.